### PR TITLE
Adjust SubscriptionChange items-quantity

### PR DESCRIPTION
**Pull request by lucasxciv**
## Summary

Adjust SubscriptionChange `items[].quantity`.

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

**Comment by lucasxciv** [comment]
@Rebilly/php @Rebilly/js please review

**Comment by lucasxciv** [comment]
@Rebilly/php please review :)

File: openapi/components/schemas/SubscriptionChange.yaml [hunk 1]
```diff
@@ -15,7 +15,6 @@ properties:
         To remove an item, include the items array and exclude the items you want to remove.
       required:
         - plan
-        - quantity
```
Comment (ashkarpetin) [inline openapi/components/schemas/SubscriptionChange.yaml:18]: Why it is not required?

Comment (lucasxciv) [reply openapi/components/schemas/SubscriptionChange.yaml:18]: For metered plans this is not required.

**Review by ashkarpetin — APPROVED** [review]

**Review by i-yasko — APPROVED** [review]

**Pull request merged**